css/custom.css

```diff
@@ -134,4 +134,24 @@ footer a:hover{
 
 .icon{
   padding-right: 5px;
-}
\ No newline at end of file
+}
+
+#myBtn {
```

Comment:
It would be better if you can name it like `backButton` :)

Reply:
I renamed it back-to-top button